### ISSUE TITLE
Test script should use local Vagrantfile

### ISSUE THREAD

**Issue by brikis98** [brikis98/docker-osx-dev#9]
Instead of downloading the latest in master. Otherwise, the file being tested is out of sync with the commit. Should probably add a `-f` flag to `docker-osx-dev init`.

**Comment by brikis98** [comment]
Depends on #7

**Comment by brikis98** [comment]
No longer relevant as docker-osx-dev no longer uses Vagrant.
